app/scripts/directives/raml-editor-import-button.js

```diff
@@ -8,7 +8,10 @@
     ) {
       return {
         restrict: 'E',
-        template: '<span role="new-button" ng-click="importFile()"><i class="fa fa-cloud-download"></i> Import</span>',
+        template:
+          '<li role="import-button" ng-click="importFile()">' +
+            '<a><i class="fa fa-cloud-download"></i> Import</a>' +
```

Comment:
Did you leave that space before import on purpose? Probably it would be a good idea to make it non-breaking: `&nbsp;`